A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_BLEED.html

```diff
@@ -6,255 +6,260 @@
-			<line class="st5" x1="120.65" y1="317.54" x2="120.65" y2="245.04" />
-			<line class="st5" x1="120.22" y1="276.29" x2="343.22" y2="276.29" />
-			<line class="st5" x1="374.22" y1="276.54" x2="480.22" y2="276.54" />
-			<line class="st5" x1="299.34" y1="276.54" x2="299.34" y2="330.42" />
-			<line id="apu-connecting-line" class="st5" style="stroke:#db7200" x1="299.09" y1="360.54" x2="299.09" y2="386.54" />
-			<line class="st5" x1="480.22" y1="317.54" x2="480.22" y2="244.83" />
-			<line class="st5" x1="300.2" y1="126.8" x2="300.23" y2="103.53" />
-			<text transform="matrix(1 0 0 1 300.5466 403.3845)" class="st1 st2 st6">APU</text>
-			<text transform="matrix(1 0 0 1 299.5466 146.3844)" class="st1 st2 st6">RAM</text>
-			<text id="left-pack-out" style="font-size: 24px;" transform="matrix(1 0 0 1 120.5466 96.3844)" class="st7 st2 st6">xx</text>
-			<text id="right-pack-out" style="font-size: 24px;" transform="matrix(1 0 0 1 480.5466 100.3844)" class="st7 st2 st6">xx</text>
-			<text id="right-pack-in" style="font-size: 24px;" transform="matrix(1 0 0 1 480.5466 167.3844)" class="st7 st2 st6">xx</text>
-			<text id="left-pack-in"  style="font-size: 24px;"transform="matrix(1 0 0 1 121.5466 167.3844)" class="st7 st2 st6">xx</text>
-			<text id="eng1-bleed-psi" transform="matrix(1 0 0 1 119.5466 332.3845)" class="st7 st2 st6">xx</text>
-			<text id="eng1-bleed-tmp" transform="matrix(1 0 0 1 118.5466 358.3845)" class="st7 st2 st6">XXX</text>
-			<text id="eng2-bleed-psi" transform="matrix(1 0 0 1 480.5466 332.3845)" class="st7 st2 st6">xx</text>
-			<text id="eng2-bleed-tmp" transform="matrix(1 0 0 1 479.5466 358.3845)" class="st7 st2 st6">XXX</text>
-			<text transform="matrix(1 0 0 1 170.5466 332.3845)" class="st8 st2 st6">PSI</text>
-			<text transform="matrix(1 0 0 1 124.5466 487.3845)" class="st1 st2 st6">IP</text>
-			<text transform="matrix(1 0 0 1 485.5466 487.3845)" class="st1 st2 st6">IP</text>
-			<text transform="matrix(1 0 0 1 209.5466 487.3845)" class="st1 st2 st6">HP</text>
-			<text transform="matrix(1 0 0 1 390.5466 487.3845)" class="st1 st2 st6">HP</text>
-			<text transform="matrix(1 0 0 1 432.5466 332.3845)" class="st8 st2 st6">PSI</text>	
-			<text transform="matrix(1 0 0 1 242.5466 314.3845)" class="st1 st2 st6">GND</text>
-			<text transform="matrix(1.22 0 0 1 300.5466 165.3846)" class="st1 st2 st6">AIR</text>
-
-			<!--engine numbers-->
-			<text transform="matrix(1 0 0 1 66.1666 414.9644)" class="st1 st2 st6">1</text>
-			<text transform="matrix(1 0 0 1 546.1666 414.9644)" class="st1 st2 st6">2</text>	
+			<!--title-->
 			
-			<!--LO/HI next to packs-->
-			<text transform="matrix(1 0 0 1 411.1766 214.3244)" class="st1 st2 st6">L</text>
-			<text transform="matrix(1 0 0 1 420.3266 214.3244)" class="st1 st2 st6">O</text>
-			<text transform="matrix(1 0 0 1 543.1766 214.3244)" class="st1 st2 st6">HI</text>
+			<text x="63" y="33" t class="st1 s2 st3">BLEED</text>
```

Comment:
```suggestion
			<text x="63" y="33" class="st1 s2 st3">BLEED</text>
```
Pretty sure this shouldn't be there, sorry I didn't find it earlier